my-app/src/app/app.module.ts

```diff
@@ -0,0 +1,30 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { HeaderComponent } from './components/header/header.component';
+import { FooterComponent } from './components/footer/footer.component';
+import { CoursesPageComponent } from './components/courses-page/courses-page.component';
+import { FakeLogoComponent } from './components/fake-logo/fake-logo.component';
+import { CoursesListComponent } from './components/courses-page/courses-list/courses-list.component';
+import { CourseItemComponent } from './components/courses-page/courses-list/course-item/course-item.component';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    HeaderComponent,
```

Comment:
create separate modules for each component and put dumb components into a shared folder. The course list is a feature module so it can be placed into `app` folder. Your structure should look similar to this. The components amount may differ, please do not copy components from the picture, just follow the structure with your components.

![image](https://user-images.githubusercontent.com/26470573/97272654-e7bc9e00-183a-11eb-9d38-6c9496d4e2d0.png)

Reply:
Fixed!